manpage/wf-recorder.1.scd

```diff
@@ -0,0 +1,99 @@
+wf-recorder(1)
```

Comment:
We already have a manpage, the audio options should be added to it instead of creating a new manpage.

Reply:
Ah, it's from the prev PR I didn't realize there was a new one